src/rebar_utils.erl

```diff
@@ -792,7 +798,8 @@ vcs_vsn_cmd(_, _, _) ->
     unknown.
 
 cmd_vsn_invoke(Cmd, Dir) ->
-    {ok, VsnString} = rebar_utils:sh(Cmd, [{cd, Dir}, {use_stdout, false}]),
+    ErrorOpt = {abort_on_error, "vsn cmd in .app.src failed"},
```

Comment:
It would be good to add the actual filename or path here as well. Otherwise, if you have many apps its up to the user to find the one with the bad vsn command